packages/elvis/CHANGELOG.json

```diff
@@ -2,6 +2,19 @@
   "$schema": "../components/changelogSchema.json",
   "name": "elvis",
   "content": [
+    {
+      "version": "19.0.0",
+      "date": "May 31, 2024",
+      "changelog": [
+        {
+          "type": "breaking_changes",
+          "changes": [
+            "Updated internal <code>@elvia/elvis-typography</code> dependency to <code>^3.0.0</code>"
+          ],
+          "components": [{ "displayName": "Typography", "url": "https://design.elvia.io/brand/typography" }]
+        }
```

Comment:
I don't think this helps our users understand what's changed. I would instead make what you've written here to be a patch, and add a breaking_change entry and describe pretty much the same as what you did in the elvis-typography changelog. 
If there are any changes that our users need to do, add a "fixes"-entry too 😄